scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java

```diff
@@ -750,7 +838,7 @@ public void onResponseReceived(Request request, Response response) {
                 int bucketSize = buckets.size();
                 for (int i = 0; i < bucketSize; i++) {
                     JSONObject bucket = buckets.get(i).isObject();
-                    String bucketName = bucket.get("name").isString().stringValue();
+                    String bucketName = bucket.get(KEY_OF_NAME).isString().stringValue();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "bucketName" which hides the field declared at line 245. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)